app/ante/eth_test.go

```diff
@@ -38,6 +43,7 @@ func (suite AnteTestSuite) TestEthSigVerificationDecorator() {
 			false,
 		},
 		{"successful signature verification", signedTx, false, true},
+		{"invalid, not replay-protected", unprotectedTx, false, false},
```

Comment:
let's create a case where the unprotected tx passes and the parameter is set to `false`